### ISSUE TITLE
Best format input file for separation?

### ISSUE THREAD

**Issue by MaxiReggi** [facebookresearch/demucs#170]
Hi! Is there any recommended format for the input file to be separated? Maybe the model would do a better job for some particular format. I use wav,  but maybe with a compressed MP3 file the model would has a better performance.

Thanks!
Cheers.

**Comment by adefossez** [comment]
Good point, actually Demucs is trained on MusDB, where the audio is coded as AAC. AAC removes everything above 16 or 18kHz, so the model is never trained with those frequencies. Maybe when fed with audio that actually contains them, it messes it up a bit.
If you have an AAC compatible ffmpeg installation, this should be straightforward to test.

If there is an impact I could try adding a lowpass to match AAC frequency cutoff.
